### PR TITLE
HDDS-2573. Handle InterruptedException in KeyOutputStream.

### PR DESCRIPTION
**Pull request by avijayanhwx**
## What changes were proposed in this pull request?

https://sonarcloud.io/project/issues?id=hadoop-ozone&issues=AW5md-m5KcVY8lQ4ZsAc&open=AW5md-m5KcVY8lQ4ZsAc

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-2573

## How was this patch tested?
Simple change, no testing done.

### CODE REVIEW

File: hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyOutputStream.java [hunk 1]
```diff
@@ -364,6 +364,7 @@ private void handleRetry(IOException exception, long len) throws IOException {
       try {
         Thread.sleep(action.delayMillis);
       } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
```
Comment (cxorm) [inline hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyOutputStream.java]: LGTM.
I build it and work well.
Waiting the retest.

**Comment by anuengineer** [comment]
The acceptance tests are timing out. This happened twice in a row in the github actions. I will run tests locally on my box and if it works I will commit these changes.

**Comment by elek** [comment]
Yes, this is timeout is a known problem and I try to identify the root cause. Unfortunately it's flaky, hard to reproduce. I triggered a new build and will commit this patch if it works.

**Comment by anuengineer** [comment]
@elek  Thank you, that retrigger worked well. You have golden hands. I have just committed the change to master.

**Comment by anuengineer** [comment]
@cxorm and @elek  Thanks for reviews and help with this patch. I have committed this to the master branch.

**Pull request merged**